pom.xml

```diff
@@ -123,8 +130,8 @@
         <version>1.1.0</version>
         <configuration>
           <cdapArtifacts>
-            <parent>system:cdap-data-pipeline[6.1.0-SNAPSHOT,7.0.0-SNAPSHOT)</parent>
-            <parent>system:cdap-data-streams[6.1.0-SNAPSHOT,7.0.0-SNAPSHOT)</parent>
+            <parent>system:cdap-data-pipeline[6.1.2-SNAPSHOT,7.0.0-SNAPSHOT)</parent>
```

Comment:
should be 6.1.1

Reply:
We tried with 6.1.1 and there was no method getRawProperties(), is this added on 6.1.2?